modules/ztp-enabling-workload-partitioning-sno.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_content-type: CONCEPT
+[id="ztp-workload-partitioning-sno_{context}"]
+= Workload partitioning in {sno} with GitOps ZTP
+
+Workload partitioning configures {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved number of host CPUs.
+
+To configure workload partitioning with GitOps ZTP, you specify cluster management CPU resources using the `cpuset` field of the `SiteConfig` CR and the `reserved` field of the group `PolicyGenTemplate` CR.
+The GitOps ZTP pipeline uses these values to populate the required fields in the workload partitioning `MachineConfig` CR (`cpuset`) and the `PerformanceProfile` CR (`reserved`) that configure the {sno} cluster.
+
+[NOTE]
+====
+For maximum performance, ensure that the `reserved` and `isolated` CPU sets do not share CPU cores across NUMA zones.
+====
+
+* The workload partitioning `MachineConfig` CR pins the {product-title} infrastructure pods to a defined `cpuset` configuration.
+* The `PerformanceProfile` CR pins the systemd services to the reserved CPUs.
+
+[IMPORTANT]
+====
+The value for the `reserved` field specified in the `PerformanceProfile` custom resource (CR) must match the `cpuset` field in the workload partitioning `MachineConfig` CR.
```

Comment:
Looks like the first spelling-out of "custom resource (CR)" for the module is down here now. Maybe swap it with the first "CR" from earlier ("`SiteConfig` CR")?